app/controllers/users_controller.rb

```diff
@@ -39,7 +38,24 @@ def edit
 	end
 
 	def update
-		@user = User.create(params)
+		user = User.find params[:id]
+	end
+
+	def save_coordinates
+		user = User.find params[:id]
+		# TODO - get location from cookie or phone location or ?
+		lat_long = "[47.6097, 122.3331]"
```

Comment:
nice this is great, eventually we can hit this route from the mobile app to update routes!